x-pack/plugins/siem/index.ts

```diff
@@ -40,6 +41,19 @@ export function siem(kibana: any) {
           url: `/app/${APP_ID}`,
         },
       ],
+      uiSettingDefaults: {
+        [DEFAULT_INDEX_KEY]: {
+          name: i18n.translate('xpack.siem.uiSettings.defaultIndexLabel', {
+            defaultMessage: 'Default index',
+          }),
+          value: ['auditbeat-*', 'filebeat-*', 'metricbeat-*', 'packetbeat-*', 'winlogbeat-*'],
```

Comment:
I think this should not say `metricbeat-*`.

Reply:
I agree, thank you.